### ISSUE TITLE
Python 3.6 support for PHCpy

### ISSUE THREAD

**Issue by berceanu** [janverschelde/PHCpack#9]
Does PHCpy support Python 3.6.1 yet?

**Comment by janverschelde** [comment]
Yes, there is support for Python 3, see 
https://github.com/janverschelde/PHCpack/tree/master/src/Python/PHCpy3

**Comment by berceanu** [comment]
Yes, I have seen that, but I have only found references to Python 3.5 in the docs, so I was wondering if it also works with 3.6.

**Comment by janverschelde** [comment]
Yes, phcpy works with python 3.6, both on Mac OS X and Linux.

**Comment by berceanu** [comment]
That's great. I am now trying to create a docker container based on [the Anaconda container](https://github.com/ContinuumIO/docker-images/tree/master/anaconda3) that would include PHCpy. I can share it with the community afterwards if you think this could be useful.
